l10n/pl/add-on.properties

```diff
@@ -2,26 +2,139 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-initiate_call_button_label2=Gotowi do rozmowy?
+clientSuperShortname=Firefox Hello
```

Comment:
Why? For some wordplay that totally doesn't work in Polish "Firefox Hello" should work much better.